### DISCUSSION TITLE
Audible Favourites

### DISCUSSION THREAD

**Discussion by dmenniss** [punxaphil/custom-sonos-card#532] [Q&A] [answered]
Is there a way to favourite audible audio books so they are displayed in the media browser?
I have added a couple of books in my Sonos app but they don't then display in the media browser. Is there a way of doing this?

Thanks for any help.

**Comment by punxaphil** [comment]
Did you add them to favorites in the sonos app?

**Reply by dmenniss** [reply]
Hi, Thanks for the response. 

Yes I have marked it as a favourite in the Sonos app and it displays in the app under favourites. Then it doesn't appear in the HA Sonos Card under favourites and also I can't browse to it. It is as if the type of media is filtered out.

Many Thanks

**Reply by punxaphil** [reply]
does it work in the ha built in media browser?

**Reply by dmenniss** [reply, marked as answer]
It doesn't appear there either so I assume from that it means it is a general HA issue?

Thanks. Looks like I need to raise with HA. 

Thanks
